jest.config.js

```diff
@@ -21,4 +21,4 @@ module.exports = {
   reporters: [
     'default',
   ]
```

Comment:
Not perfect yet but imo a really good first step toward having a decent test structure.